examples/darwin-framework-tool/commands/clusters/ClusterCommandBridge.h

```diff
@@ -97,7 +95,7 @@ class ClusterCommand : public ModelCommand {
                                      }];
 
             if (mRepeatDelayInMs.HasValue()) {
-                chip::test_utils::SleepMillis(mRepeatDelayInMs.Value());
+                [NSThread sleepForTimeInterval:mRepeatDelayInMs.Value() / 1000];
```

Comment:
This will do the wrong thing if `mRepeatDelayInMs.Value() < 1000`, no?  As in, it will sleep for 0.

This should case `mRepeatDelayInMs.Value()` to `double` before doing the division, I would think.

Reply:
Updated. Thanks